components/raftstore/src/store/fsm/peer.rs

```diff
@@ -591,8 +592,8 @@ where
 
                     let req_size = cmd.request.compute_size();
                     if self.ctx.cfg.cmd_batch
-                        && cmd.extra_opts.disk_full_opt == DiskFullOpt::NotAllowedOnFull
                         && self.fsm.batch_req_builder.can_batch(&cmd.request, req_size)
+                        && cmd.extra_opts.disk_full_opt == DiskFullOpt::NotAllowedOnFull
```

Comment:
Seems the `disk_full_op` has nothing to do with cmd batch?

Reply:
Normal requests will be rejected in `propsoe` if the store's disk is full. So here we avoid to merge them, so that only some special operations will be allowed.

Reply:
Got it. Would you mind to add comments?